webapp/src/components/DonationsDashboard/DonationsDashboard.js

```diff
@@ -23,8 +23,12 @@ import QRCode from 'qrcode.react'
 import QrReader from 'react-qr-scanner'
 import { useTranslation } from 'react-i18next'
 import Snackbar from '@material-ui/core/Snackbar'
+import useMediaQuery from '@material-ui/core/useMediaQuery'
+import { useTheme, makeStyles } from '@material-ui/core/styles'
+import KeyboardBackspaceIcon from '@material-ui/icons/KeyboardBackspace'
 
 import { PROFILE_QUERY, TRANSFER_MUTATION } from '../../gql'
+import { Drawer } from '@material-ui/core'
```

Comment:
Please use `import Drawer from '@material-ui/core/Drawer'` instead
Remember to place import right up